README.md

```diff
@@ -275,6 +263,34 @@ More info [here](https://github.com/microsoft/vscode-eslint#mono-repository-setu
 
 Building a docker image, read the [docker doc](./docs/docker/docker.md).
 
+
+## 8. Releases, changelogs and versioning
+
+_Apps_ and _proxies_ use automatic semantic versions and are released using [Release Please](https://github.com/googleapis/release-please).
+
+> Release Please is a GitHub Action that automates releases for you. It will create a GitHub release and a GitHub Pull Request with a changelog based on conventional commits.
+
+Each time you merge a pull request, the release-please-action will create or update a "Release PR" with the changelog and the version bump related to the changes. To create a new release for an app, this release PR is merged, which creates a new release with release notes and a new tag. This tag will be picked by Azure pipeline and trigger a new deployment to staging. From there, the release needs to be manually released to production.
+
+### Conventional commits
+
+Use conventional commits to ensure that the changelogs are generated correctly.
+
+### Configuration
+
+The release-please workflow is located in the [release-please.yml](./.github/workflows/release-please.yml) file.
+
+The configuration for release-please is located in the [release-please-config.json](./release-please-config.json) file. See vatious options in the [release-please docs](https://github.com/googleapis/release-please/blob/main/docs/manifest-releaser.md).
+
+The manifest file is located in the [release-please-manifest.json](./.release-please-manifest.json) file. When adding a new app, add it to this file with the current version of the app. After this, release-please will keep track of versions with this while.
+
+
+### Troubleshoting release-please
+
+If a new release PR is not created, there's propably one of the older release PR's still in pending state: make sure there's no open release pr already. Also see if any of the closed PR's are labeled with `autorelease: pending` and change their label to `autorelease: tagged`. Then go and re-run the last merge workflow to trigger the release action.
```

Comment:
> Each time you merge a pull request, the release-please-action will create or update a "Release PR"
&
> If a new release PR is not created, there's propably one of the older release PR's still in pending state: make sure there's no open release pr already.

Shouldn't the release-please-action update an open pending release PR?

Reply:
Yes, this was badly worded, the intention is to say that _if_ you were expecting a *new* PR, then you should make sure there's no pending PR's, neither open nor closed; I'll rephrase it.